tool_quality_checks/correct_import_file.py

```diff
@@ -129,6 +129,15 @@ def try_fix_geometry(geometry):
     return geometry, geometry.IsValid()
 
 
+def correct_layer_name_length(layer_name):
+    logger.info("check - Name length")
+    if len(layer_name) + 10 > 64:
+        logger.warning("laagnaam te lang, 50 characters max.")
+        logger.warning("formatting naar 50 met deze naam: %s" % layer_name[:50])
+        layer_name = layer_name[:50]
```

Comment:
- De ":50" maar 1 keertje doen.
- Suggestie om de melding wat duidelijker te maken.

```suggestion
        layer_name = layer_name[:50]
        logger.warning("Hij is nu ingekort tot '%s'." % layer_name)
```